cmd/k8s-operator/deploy/chart/templates/deployment.yaml

```diff
@@ -68,6 +76,10 @@ spec:
               value: /oauth/client_id
             - name: CLIENT_SECRET_FILE
               value: /oauth/client_secret
+            {{- if .Values.operatorConfig.port.enabled }}
+            - name: PORT
```

Comment:
@jaxxstorm this will need changing to `TS_PORT`, see https://github.com/tailscale/tailscale/pull/14005